packages/core/saved-objects/core-saved-objects-migration-server-internal/src/model/helpers.ts

```diff
@@ -65,6 +65,7 @@ export function mergeMigrationMappingPropertyHashes(
   return {
     ...targetMappings,
     _meta: {
+      ...targetMappings._meta,
```

Comment:
There's no unit tests for this `mergeMigrationMappingPropertyHashes` helper. We should add some (and assert the behavior of this change/fix)

Reply:
We should do it on the upstream PR.

But first I wanted to check whether this `mergeMigrationMappingPropertyHashes` operation is still necessary.
If we want to use these hashes as a basis to construct the `indexTypesMap`, I'm afraid the _merge_ can be problematic. @rudolf WDYT?

Reply:
We use these in `checkTargetMappings` as a way to determine if mappings are up to date. If we don't merge the _meta it means there could be plugins which have applied some mappings that we don't know about. This would cause us to go to `UPDATE_TARGET_MAPPINGS_PROPERTIES` which would be unecessary but not a problem. So I don't see an obvious problem right now.